kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverVersions.kt

```diff
@@ -17,7 +17,12 @@ public object KoverVersions {
     /**
      * JaCoCo coverage tool version used by default.
      */
-    public const val JACOCO_TOOL_DEFAULT_VERSION = "0.8.12"
+    public const val JACOCO_TOOL_DEFAULT_VERSION = "0.8.13"
+
+    /**
+     * JaCoCo coverage tool minimal supported version.
+     */
+    public const val JACOCO_TOOL_MINIMAL_VERSION = "0.8.13"
```

Comment:
What dictates this restriction? At first glance, it seems like it's only missing `fragments` property that stops us from using earlier version. And if I got it correctly, we may ignore it for now.

Reply:
Yes, and this way we can guarantee support for inline functions. 
I do not know the reasons why older versions of JaCoCo might be needed.